general/README.md

```diff
@@ -257,3 +257,16 @@ function getList(
 	parameter3
 ) {}
 ```
+
+
+## Spaces
+When working with variables between parantheses, always seperate them by adding spaces. 
```

Comment:
separate 😇

Reply:
And aLways should be always. 👮

Reply:
They are called curly braces instead of parantheses ;)

Reply:
parentheses 😇